### PR TITLE
fix(Signup): send last 10 items in utm cookies

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Change in signup when create user, send the last 10 utm cookies to fix the problem when the utm cookies are a lot.

image

### CODE REVIEW

File: src/components/Signup/Signup.js [hunk 1]
```diff
@@ -172,7 +172,7 @@ const Signup = function ({
       utm_campaign: utmCampaign,
       utm_medium: utmMedium,
       utm_term: utmTerm,
-      utm_cookies: utmCookies,
+      utm_cookies: utmCookies.slice(-10),
```
Comment (andresmoschini) [inline src/components/Signup/Signup.js:175]: Si hay menos de 10 no rompe, no?

Comment (andresmoschini) [reply src/components/Signup/Signup.js:175]: Tampoco utmCookies puede ser null, correcto?

Comment (fcoronelmakingsense) [reply src/components/Signup/Signup.js:175]: image

Comment (fcoronelmakingsense) [reply src/components/Signup/Signup.js:175]: @andresmoschini

**Comment by andresmoschini** [comment]
🚀 

Si `utmCookies` no puede ser null, dale para adelante...

**Pull request merged**